Kudu.Services/Diagnostics/ProcessController.cs

```diff
@@ -274,6 +276,35 @@ public HttpResponseMessage MiniDump(int id, int dumpType = 0, string format = nu
             }
         }
 
+        [HttpPost]
+        public HttpResponseMessage TakeCrashDump(int id, bool includeMemory = false)
+        {
+            using (_tracer.Step("ProcessController.TakeCrashDump"))
+            {
+                try
+                {
+                    var process = GetProcessById(id);
+                    var exe = new Executable(ResolveProcDumpPath(), _environment.CrashDumpsPath, TimeSpan.MaxValue);
+                    var memoryFlag = includeMemory ? "-ma" : string.Empty;
+
+                    Task.Run(() => exe.ExecuteReturnExitCode(_tracer, s => _tracer.Trace(s), s => _tracer.TraceError(s), $"-accepteula -t {process.Id} -e -g {memoryFlag}"));
+
```

Comment:
I would think it should be lower (maybe 5).   Simply give error on POST crashdump, when there are already 5 of them, perhaps.